ChiamakaObitube/PropertyPro-Lite
# #166712554 Build out UI for User to View All Properties

**What does this PR do?**
This PR creates a view all properties page for the product

**Description of Task to be completed?**
Have the following working: create a view all property advert page

**How should this be manually tested?**
The following steps can be used:
Clone the repository, cd into the UI folder and  see the view all property advert.html file

**Any background context you want to provide?**
User should ensure he has access to the internet

**What are the relevant pivotal tracker stories?**
[#166712554](https://www.pivotaltracker.com/story/show/166712554)
